Osoti12exercises.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
<p>This works! You can also use a Python package to get an exact value for pi. We can talk a bit about installing and using packages tomorrow. A package is a set of functions or information that you can use in your own code. We use them a lot! You can Google for a package that has pi and find that the package 'math' as a value for pi included.  Here's an example of how you might use it. </p>

<pre class="ql-syntax" spellcheck="false">import math

pi_value = math.pi
print(pi_value)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>